c/array-programs/Makefile

```diff
@@ -0,0 +1,6 @@
+LEVEL := ../
+
+CLANG_WARNINGS := \
+	-Wno-error=uninitialized \

```

Comment:
Is this `no-error=uninitialized` really necessary? It would be better to fix the benchmarks instead of suppressing the warning, because this is probably undefined behavior.

Reply:
Arrays in the programs are to be left uninitialised. Therefore, "no-error=uninitialized" is necessary.

Reply:
Oh, but having undefined behavior is not allowed, and this indeed occurs in these benchmarks.

You need to ensure that all array cells that are read have been initialized before.
Note that in files like the `copysome` programs, this even means you need to initialize at least some of the `a2` cells. Fixing all compiler warnings is not enough, because compilers are not guaranteed to find all undefined behavior.

If you want non-deterministic values, you need to call `__VERIFIER_nondet_int` in a loop.

So please ensure that there is no undefined behavior and remove the compiler warning.